programs/benchmark/Benchmark.cpp

```diff
@@ -469,7 +469,7 @@ class Benchmark : public Poco::Util::Application
                 for (const auto & conn : connections)
                 {
                     if (!connection_description.empty())
-                        connection_description += ",";
+                        connection_description += ", ";
```

Comment:
This will make it impossible to distinguish list of hosts from other columns, since other columns is printed via `, ` too, i.e.:

Before:

```
127.1:9000,127.2:9000, queries 2, QPS: 48.036, RPS: 48.036, MiB/s: 0.000, result RPS: 48.036, result MiB/s: 0.000.
```

After:

```
127.1:9000, 127.2:9000, queries 2, QPS: 48.036, RPS: 48.036, MiB/s: 0.000, result RPS: 48.036, result MiB/s: 0.000.
```

Reply:
I don't like lack of whitespace after comma :crying_cat_face: